runtime/Java/src/org/antlr/v4/runtime/atn/PredictionContext.java

```diff
@@ -269,33 +277,42 @@ public static PredictionContext mergeSingletons(
 	 * {@link EmptyPredictionContext#Instance}. In the following diagrams, the symbol {@code $} is used
 	 * to represent {@link EmptyPredictionContext#Instance}.
 	 *
-	 * <h2>Local-Context Merges</h2>
+	 * <h4>Local-Context Merges</h4>
 	 *
-	 * <p>These local-context merge operations are used when {@code rootIsWildcard}
-	 * is true.</p>
+	 * <p>These local-context merge operations are used when {@code rootIsWildcard} is true.</p>
 	 *
 	 * <p>{@link EmptyPredictionContext#Instance} is superset of any graph; return {@link EmptyPredictionContext#Instance}.<br>
-	 * <embed src="images/LocalMerge_EmptyRoot.svg" type="image/svg+xml"/></p>
+	 * <img src="images/LocalMerge_EmptyRoot.svg"
+	 *      alt="Case where at least one of a or b is empty, subcase 1."/>
+	 * </p>  
 	 *
 	 * <p>{@link EmptyPredictionContext#Instance} and anything is {@code #EMPTY}, so merged parent is
 	 * {@code #EMPTY}; return left graph.<br>
-	 * <embed src="images/LocalMerge_EmptyParent.svg" type="image/svg+xml"/></p>
+	 * <img src="images/LocalMerge_EmptyParent.svg"
+	 *      alt="Case where at least one of a or b is empty, subcase 2."/>
+	 * </p>  
 	 *
 	 * <p>Special case of last merge if local context.<br>
-	 * <embed src="images/LocalMerge_DiffRoots.svg" type="image/svg+xml"/></p>
-	 *
-	 * <h2>Full-Context Merges</h2>
+	 * <img src="images/LocalMerge_DiffRoots.svg"
+	 *      alt="Case where at least one of a or b is empty, subcase 3."/>
```

Comment:
Hi. is this the correct alt tag? I can't tell just by looking at the diff.

Reply:
Sorry for the late reply. Yes it is, in the sense as "it describes what the image shows"

Reply:
Regarding the 404 on: https://www.antlr.org/api/Java/

It is the search box which fails. 

I also vaguely remember looking for a class but not finding it. It could be javadoc did not generate the HTML page.

Reply:
Regarding the question "did you try to run mvn to get javadoc" - no, I did not think about that.

Regarding the "embed" tag, it turned out that the "javadoc" tool rejects it. However, the "img" tag is allowed. Both "embed" and "img" are HTML content for javadoc according to this page: 

- https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html

where it says "HTML constructs should be written in HTML 5"

So I went with the "img" tag, which empirically works (tested locally) but both the _img_ tag and the _embed_ tag are correct HTML5:

- https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element
- https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-embed-element

"Tutorialspoint" for one says all are fine:

- https://www.tutorialspoint.com/Should-I-use-img-object-or-embed-for-SVG-files

Reply:
All right, I missed that these instructions exist:

https://github.com/antlr/antlr4/blob/master/doc/releasing-antlr.md#javadoc-for-runtime-and-tool

Reply:
Let me check the last comment later today, I will have to do some garage work first.